### ISSUE TITLE
[APP SUBMITTED]: <sickbeard.tv.TVEpisode object at 0x97a002c>

### ISSUE THREAD

**Issue by TehWardy** [SiCKRAGETV/sickrage-issues#706]
### INFO
Python Version: **2.7 (r27:82500, Aug 21 2010, 07:52:47) 
[GCC 4.1.3 20070929 (prerelease) (Ubuntu 4.1.2-16ubuntu2)]**
Operating System: **Linux-3.4.6-x86_64-with-glibc2.4**
Branch: **develop**
Commit: SiCKRAGETV/SickRage@fbcadb278054808639884dca5804d544bbc0a675
### ERROR
```
SEARCHQUEUE-BACKLOG-153021 :: [Usenet-Crawler] :: Error while searching Usenet-Crawler, skipping: error The Walking Dead - 5x10 - Them
location: 
description: The group feels beaten after living life on the road, but must continue to trudge along in spite of the changes they have incurred.
subtitles: 
subtitles_searchcount: 0
subtitles_lastsearch: 0001-01-01 00:00:00
airdate: 735644 (2015-02-15)
hasnfo: False
hastbn: False
status: 3

```
---
_STAFF NOTIFIED_: @SiCKRAGETV/owners @SiCKRAGETV/moderators

**Comment by TehWardy** [comment]
I have no idea why this occurs, maybe its related to the parsing issue but figured it might be worth making a note of it here in case this info provides some additional insight in to what's going on.

**Comment by fernandog** [comment]
did it happen again? did you succeeded download it ?

**Comment by TehWardy** [comment]
With my limited knowledge of linux I would suggest adding more logging calls and have the bug reporting tool include them for errors like this.
Maybe that could help more than I can.

**Comment by fernandog** [comment]
@TehWardy still an issue?

**Comment by TehWardy** [comment]
I haven't seen it since applying the parse fix from #707 #708 bugs ... 

Maybe my theory was correct and this information was just related to that.

Lets close this and if it comes back I can always report it again hopefully with a bit more detail on the problem.
